src/deluge/gui/l10n/seven_segment.json

```diff
@@ -500,7 +500,17 @@
         "STRING_FOR_NOTE_EDITOR_VELOCITY": "VELO",
         "STRING_FOR_NOTE_EDITOR_PROBABILITY": "PROB",
         "STRING_FOR_NOTE_EDITOR_ITERANCE": "ITER",
-        "STRING_FOR_NOTE_EDITOR_FILL": "FILL",   
+        "STRING_FOR_NOTE_EDITOR_FILL": "FILL",  
+
+        "STRING_FOR_ITERANCE_DIVISOR": "DIVI",
+        "STRING_FOR_ITERATION_STEP_1": "IT 1",
```

Comment:
We have print formatting - one string for IT or Iteration plus sprintf will probably be more readable

Reply:
But do our current Toggle menus accept sprintf strings?. This is the constructor for IteranceStepToggle:

	IteranceStepToggle(l10n::String newName, l10n::String title, uint8_t newIndex) : Toggle(newName, title) {
		this->index = newIndex;
	}
	
how do I convert it to accept the "IT " label and append there the (newIndex+1) value as sprintf?

Reply:
No but you can make a new menu class that appends a number - check the envelopes or oscillators, we ideally dont want 8 menu entries either

Reply:
@soymonitus I'm going to work on the menu refactoring so don't worry about this part. If you can look at Mark's other feedback that would be great.

Thanks!